src/test/java/uk/nhs/prm/e2etests/performance/load/RoundRobinPoolTest.java

```diff
@@ -10,35 +10,35 @@
 
 class RoundRobinPoolTest {
 
-    private List<Integer> integers = of(1, 2, 3);
+    private final List<Integer> integers = of(1, 2, 3);
 
     @Test
-    public void shouldGiveFirstItemFirst() {
-        var pool = new RoundRobinPool<>(integers);
+    void shouldGiveFirstItemFirst() {
+        RoundRobinPool<Integer> pool = new RoundRobinPool<>(integers);
 
-        var first = pool.next();
+        Integer first = pool.next();
 
         assertThat(first).isEqualTo(1);
     }
 
     @Test
-    public void shouldGiveSecondItemSecond() {
-        var pool = new RoundRobinPool<>(integers);
+    void shouldGiveSecondItemSecond() {
+        RoundRobinPool<Integer> pool = new RoundRobinPool<>(integers);
 
         pool.next();
-        var second = pool.next();
+        Integer second = pool.next();
 
         assertThat(second).isEqualTo(2);
     }
 
     @Test
-    public void shouldLoopRoundToFirstItemAgainAfterPoolExhausted() {
-        var poolOfThree = new RoundRobinPool<>(integers);
+    void shouldLoopRoundToFirstItemAgainAfterPoolExhausted() {
+        RoundRobinPool<Integer> poolOfThree = new RoundRobinPool<>(integers);
 
         poolOfThree.next();
         poolOfThree.next();
-        var third = poolOfThree.next();
-        var fourth = poolOfThree.next();
+        Integer third = poolOfThree.next();
+        Integer fourth = poolOfThree.next();
```

Comment:
Can we change this to it's primitive type? 

```suggestion
        int third = poolOfThree.next();
        int fourth = poolOfThree.next();
```